EU4toV2/Source/Mappers/FactoryTypes/FactoryType.h

```diff
@@ -1,41 +1,38 @@
 #ifndef FACTORY_TYPE_H
```

Comment:
Add tests

Reply:
Look at Mappers_FactoryTypeTests, these cover this file and explain the entire stack.

Reply:
btw. On related note, I noticed VS having issues with "show all references", not listing any when I *clearly* have function calls, or listing them partially - those from within the project but not those from tests.

then I open up the tests and suddenly "show all references" works.

Reply:
https://developercommunity.visualstudio.com/content/problem/514082/vs-2019-find-all-references.html

Reply:
This one is 10 days old:
https://developercommunity.visualstudio.com/content/problem/1256537/find-all-referencesgo-to-implementation-returns-no.html

"Closed, not a bug". WTF